### DISCUSSION TITLE
How to make Modal to dynamically resize to its content size?

### DISCUSSION THREAD

**Discussion by danvln** [ant-design/ant-design#29954] [Q&A]
Hello,

I have a modal that has variable content, and I would like to resize it based on the height of its content. 

Could anyone let me know if has found a solution for this, or share any available code that would make its implementation easier?

Thanks!

**Comment by Chevindu** [comment]
For anyone stumbling upon this in future, Modal should resize according to the content height by default. 

However for width, I found the following to work as I expected.

```jsx
<Modal width="fit-content">...</Modal>
```
